src/components/progress-tracker/progress-tracker.mdx

```diff
@@ -57,61 +55,37 @@ This is an example of a large `ProgressTracker` component.
 
 ### Custom bar length
 
-The length of the bar can be set to any valid css property. This sets the width of a horizontal bar or the height of a vertical bar. The default is "256px".
+By default, the length of the component is "256px". However, you can use the `length` prop to override this with any valid css string. 
 
 <Canvas of={ProgressTrackerStories.CustomBarLength} />
 
 ### Color variants
 
-This is an example of a `ProgressTracker` component with different colors depending on progress.
+Depending on the progress or state of the component, the colour of the progress bar is automatically updated.
 
 <Canvas of={ProgressTrackerStories.ColorVariants} />
 
-### Default label values
-
-This is an example of a `ProgressTracker` component with `showDefaultLabels` set to `true`, which will render the labels as
-percentages.
-
-<Canvas of={ProgressTrackerStories.DefaultLabelValue} />
-
-It is possible to override the default `labelsPosition` to `"bottom"` or `"left"`.
-
-<Canvas of={ProgressTrackerStories.DefaultLabelValueLabelsPositionBottom} />
-
-When positioning the labels to the left, it is possible to use the `LabelWidth` prop to ensure consistent label lengths.
-This can also facilitate comparison between multiple `ProgressTrackers`.
-
-<Canvas of={ProgressTrackerStories.DefaultLabelValueLabelsPositionLeft} />
 
 ### Custom label values
 
-This is an example of a `ProgressTracker` component with `currentProgressLabel` and `maxProgressLabel` props.
+To set your own labels, you can set the `currentProgressLabel` and `maxProgressLabel` props. 
 
-<Canvas of={ProgressTrackerStories.CustomLabelValues} />
-
-It is also possible to render the component with a mix of default and custom labels by only setting one of the `currentProgressLabel`
-and `maxProgressLabel` props and optionally setting the `showDefaultLabels` prop.
-
-<Canvas of={ProgressTrackerStories.DefaultAndCustomLabelValues} />
+Note that the preposition between the current and max labels will be 'of' by default, but you can override this by setting the `customValuePreposition` prop and making use of the `of` translation key.
```

Comment:
**suggestion(non-breaking)**: this might make it clearer that consumers don't need to do both :)

```suggestion
Note that the preposition between the current and max labels will be 'of' by default, but you can override this by setting the `customValuePreposition` prop or making use of the `of` translation key.
```